### PR TITLE
Update SQLStatement log level(#21764)

### LINKED ISSUE #21764: [SQLLogger] SQLStatement log
## Feature Request

Hi，I don't need a SQLStatement, SQLStatement prints too much.

Can you add a switch to the SQLStatement of SQLLogger to control whether to print?

### PR DESCRIPTION
**Pull request by smartbetter**
Changes proposed in this pull request:

- fixes #21764. update SQLStatement log level to DEBUG.

---

Before committing this PR, I'm sure that I have checked the following options:

- [x] My code follows the code of conduct of this project.
- [x] I have self-reviewed the commit code.
- [x] I have (or in comment I request) added corresponding labels for the pull request.
- [x] I have passed maven check locally : ./mvnw clean install -B -T1C -Dmaven.javadoc.skip -Dmaven.jacoco.skip -e.
- [ ] I have made corresponding changes to the documentation.
- [ ] I have added corresponding unit tests for my changes.

### CODE REVIEW

File: infra/executor/src/main/java/org/apache/shardingsphere/infra/executor/sql/log/SQLLogger.java [hunk 1]
```diff
@@ -44,7 +44,7 @@ public final class SQLLogger {
      */
     public static void logSQL(final QueryContext queryContext, final boolean showSimple, final ExecutionContext executionContext) {
         log("Logic SQL: {}", queryContext.getSql());
-        log("SQLStatement: {}", queryContext.getSqlStatementContext().getSqlStatement());
+        log.debug("SQLStatement: {}", queryContext.getSqlStatementContext().getSqlStatement());
```
Comment (terrymanu) [inline infra/executor/src/main/java/org/apache/shardingsphere/infra/executor/sql/log/SQLLogger.java:47]: How about move it to `logNormalMode`?

**Review by terrymanu — CHANGES_REQUESTED** [review]

File: infra/executor/src/main/java/org/apache/shardingsphere/infra/executor/sql/log/SQLLogger.java (hunk 1 above)
Comment (terrymanu) [reply infra/executor/src/main/java/org/apache/shardingsphere/infra/executor/sql/log/SQLLogger.java:47]: Sorry, ShardingSphere is a public project, it should not upon to one person's requirement.

Comment (terrymanu) [reply infra/executor/src/main/java/org/apache/shardingsphere/infra/executor/sql/log/SQLLogger.java:47]: I created a new issue to discuss about remove it. #22453

Comment (smartbetter) [reply infra/executor/src/main/java/org/apache/shardingsphere/infra/executor/sql/log/SQLLogger.java:47]: > I created a new issue to discuss removing it. #22453

Okay, thanks.

**Pull request closed without merging**